docs/build/configure/_index.md

```diff
@@ -381,11 +381,42 @@ Switch to **Raw JSON** mode to configure webhooks as follows:
 
 ## Fragments
 
-Fragments are a way of sharing and managing identical {{< glossary_tooltip term_id="resource" text="resource" >}} configuration files across multiple machines.
-For example, if you have multiple machines with the same hardware, wired the same way, you can create and share a fragment and add it to any number of machines.
-When changes are made to the fragment, those changes are automatically carried to all machines that include the fragment in their config.
+You can use fragments to share identical {{< glossary_tooltip term_id="resource" text="resource" >}} configuration files across multiple machines in your fleet.
+For example, if you have multiple machines with the same camera hardware, wired the same way, you can create a fragment to configure that camera and share it easily across all of your machines, without needing to individually configure the camera component for each machine.
```

Comment:
I like this more specific example but, how about changing it to "motor" instead of "camera" because cameras are more likely to just be USB or something versus what we think of as "wired the same way"--a motor usually has several wires and a more complex description of that wiring in its config, such that it's useful to put it in a fragment.

Reply:
Great point, updating to motor, thank you!